tests/test_cli.py

```diff
@@ -0,0 +1,193 @@
+import pytest
+from pathlib import Path
+import subprocess
+
+import pytest
+from pathlib import Path
+import subprocess
+from lando_cli.cli import (
+    Config,
+    get_new_commits,
+    get_commit_patches,
+    get_commit_message,
+    detect_new_tags,
+    detect_merge_from_current_head,
+    get_current_branch,
+)
+
+
+@pytest.fixture
+def git_remote_repo(tmp_path: Path):
+    """Create a temporary bare remote Git repo."""
+    remote_repo = tmp_path / "remote.git"
+    subprocess.run(["git", "init", "--bare", remote_repo.as_posix()], check=True)
+    yield remote_repo
+
+
+@pytest.fixture
+def git_local_repo(tmp_path: Path, git_remote_repo: Path):
+    """Create a temporary local Git repo with remote set up."""
+    local_repo = tmp_path / "local"
+    local_repo.mkdir()
+
+    subprocess.run(["git", "init", local_repo.as_posix()], check=True, cwd=local_repo)
```

Comment:
nit: We could just 
```suggestion
    subprocess.run(["git", "clone", git_remote_repo.as_posix(), local_repo.as_posix()], check=True, cwd=local_repo)
```

And avoid adding the origin dance later.